org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -242,7 +258,7 @@ protected void updateQueue(U bean) throws EventException {
 				
 				if (rem == null && b.getUniqueId().equals(bean.getUniqueId())) { 
 					// Something went wrong, not sure why it does this, TODO investigate
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)